docs/index.md

```diff
@@ -54,23 +54,26 @@ This section will provide you by a quick guide to start.
 
 ### Dependencies
 This project depends on the following packages:
-1. argon2-cffi = 23.1.0
-2. argon2-cffi-bindings = 21.2.0
-3. cffi = 1.16.0
-4. numpy = 1.25.0
-5. Pillow = 10.1.0
-6. pycparser = 2.21
-7. PyQt5 = 5.15.10
-8. PyQt5-Qt5 = 5.15.2
-9. PyQt5-sip = 12.13.0
+```bash
+argon2-cffi
+argon2-cffi-bindings
+cffi
+numpy
+Pillow
+pycparser
+PyQt5
+PyQt5-Qt5
+PyQt5-sip
+```
```

Comment:
I'm actually like the order list of the requirements with each version in this list, so, any one can read the docs get insight about the dependencies of the project without any effort. My question is why you removed the versions and putted the list in `bash` snippet which is not?

Reply:
I submitted a commit to let the dependencies in a list and removed the versions because maintaining the versions here will add extra efforts beside the ones in `requirements.txt` file.